guide/daily-spending-wallet/sending.md

```diff
@@ -175,7 +175,7 @@ If no amount is provided via a payment request, manual entry should be simple an
 
 **Metadata**
 
-A transaction history is hard to make sense of when it only shows amounts, dates, and identifiers. Users should be allowed to add descriptions, tags, and other metadata to add context. This context can separately be used for helpful tools like visual spending breakdowns.
+A [transaction history]({{ '/guide/daily-spending-wallet/activity/' | relative_url }}) is hard to make sense of when it only shows amounts, dates, and identifiers. Users should be allowed to add descriptions, tags, and other metadata to add context. This context can separately be used for helpful tools like visual spending breakdowns.
 
 ## Fees
```

Comment:
We should include a carousel here that shows an image for each of the three examples we have listed. For the on-chain part we should re-focus that to be in the context of swaps as we have discussed elsewhere. 

This would also slightly reduce the overall size of this section + total page length.

Reply:
The activity page has a carousel specifically for this. I'd just make that more clear ("more on this in...").

Which on-chain part are you referring to? Don't see anything on that in this paragraph.

Reply:
I was referring to the fees section here with this comment, not the manual payment initiation section. The on-chain bit is at the bottom of that section. 

![image](https://user-images.githubusercontent.com/55287964/169199852-9fcc5b83-9b46-4006-ae57-07f79a96e772.png)

Reply:
You don't think the two images in that section already convey the fee states well enough?

Reply:
They do but the visual style of the Lightning routing fees image is a little field and looks completely different to any other image used in the reference design. Would be better to stick with using phone frames and having a carousel with three images, one high lightning routing fees, one service fees, one on-chain fees. This would flow nicer with the rest of the page which uses carousels / also make the page slightly less tall

Reply:
I intentionally went that route because it's a fairly small detail and harder to compare when you look at thee screens in a gallery. I think there are times when it's better to just focus on a single component this way. Similar to the status and recipient field visuals on the [activity page](https://bitcoin.design/guide/daily-spending-wallet/activity/#status-display).

Reply:
You could show one frame that has lightning services and network fee in the one with some text highlighting both

Reply:
The black image sort of confused me my first several times looking at it. It's not entirely clear what's going on.

Since I'm familiar with the Guide, my guess is that the top one is a basic lightning payment fee, then the second one is for a payment that requires a channel open.

What about the last one? Is this supposed to be an on-chain payment?

Having the fees divorced from their screens certainly makes me have to work harder to figure out what's going on. I get it, but only after I think about it for a moment.

3 screens with little descriptions under them would be super clear. e.g. "Above is the fee for most lightning payments", "In this situation, a payment channel is needed...", etc.